main.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_ecs_cluster_capacity_providers" "atlantis" {
+  cluster_name = var.create_ecs_cluster ? aws_ecs_cluster.atlantis[0].name : var.ecs_cluster_name
+
+  capacity_providers = ["FARGATE"]
```

Comment:
Spot would be a very logical choice for a workload that uses external persistent storage. I think that this should be possible.